### PR TITLE
chore(NA): add kibana 6.7.0

### PR DESCRIPTION
**Pull request by mistic**
This PR adds the dockerfiles for `kibana 6.7.0` as it is being done by the elasticsearch team.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,30 +3,46 @@
 ## About this Repository
 
 This repo is used to store the Dockerfile which can be used to build an
-Elasticsearch Docker image. The Dockerfile was generated from the [Elasticsearch
-repository](https://github.com/elastic/elasticsearch). Please note that **issues
-are disabled on this repo** and that all issues and PRs must be filed in the
-[Elasticsearch repository](https://github.com/elastic/elasticsearch).
+Elasticsearch and a Kibana Docker image. The Dockerfile was generated from their
```
Comment (tylersmalley) [inline README.md]: Thoughts on generalizing for the Elastic stack? I suspect all other projects to follow suit.

Comment (mistic) [reply README.md]: I thought about that @tylersmalley . I didn't get in a consensus with myself ( 😄) however maybe the best way would be just do what I did for Kibana, extending the related information for each stack product. Maybe for that paragraph we can just use the following:

```
This repo is used to store the Dockerfile which can be used to build Docker images for each product released in the stack. Those Dockerfiles were generated from the products' own repositories for which you can get the links in the sections below. Please note that **issues
are disabled on this repo** and that all issues and PRs must be filed in the products' repositories.
``` 

What do u think?

Comment (Conky5) [reply README.md]: I like the generic paragraph and separate sections for each project idea.

**Comment by mistic** [comment]
@tylersmalley @Conky5 thanks for the feedback! I've updated the README in order to have it more generic in a way it would be able to support the other products on the stack. What do u think right now?

**Pull request merged**